spec/controllers/v0/triage_teams_controller_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe V0::TriageTeamsController, type: :controller do
+  let(:id) { ENV['MHV_SM_USER_ID'] }
+
+  describe 'index' do
+    before(:each) do
+      VCR.use_cassette("triage_teams/#{id}/index") do
+        get :index, id: id
+      end
+    end
+
+    it 'sets the correlation id' do
+      expect(assigns(:mhv_correlation_id)).to eq(id)
```

Comment:
can probably just get rid of this spec once you base it off the hardcoded constant value.